site/src/sphinx/setup-installation.rst

```diff
@@ -60,3 +60,12 @@ To stop the server, use the ``bin/shutdown`` script:
     You can also tail the log file::
 
         $ ./gradlew tail
+
+Run on docker
+--------------------
```

Comment:
- Run -> Running
- docker -> Docker
- Mismatching number of hyphens - should be of same length with the section title